spec/components/question/organisation_name_component/organisation_name_component_preview.rb

```diff
@@ -0,0 +1,24 @@
+class Question::OrganisationNameComponent::OrganisationNameComponentPreview < ViewComponent::Preview
+  def organisation_name_field
+    question = OpenStruct.new(text: "Organisations R Us ",
```

Comment:
Is the trailing space deliberate? If not then think it can be removed:

```suggestion
    question = OpenStruct.new(text: "Organisations R Us",
```

Reply:
good spot, thanks @DavidBiddle